### PR TITLE
Feat: remove from playlist, empty page stylings

### PR DESCRIPTION
**Pull request by anjali1102**
- functionality of category on homepage
- remove videos & individual playlists from /playlist itself earlier it was on modal only.
- empty page stylings
- toasts added
- loader on homepage

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
code looks good to be merged

File: src/pages/SignUp/SignUp.jsx [hunk 1]
```diff
@@ -1,38 +1,63 @@
...
 import { Link } from "react-router-dom";
+import { signUpUser } from "../../store/authSlice";
+import { useNavigate } from "react-router-dom";
 import "./SignUp.css";
 
 const SignUp = () => {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
```
Comment (mehulsatardekar) [inline src/pages/SignUp/SignUp.jsx:12]: this can be combined into one usestate of object like useState({firstname:"", lastname:""}) so on

**Review by mehulsatardekar — APPROVED** [review]
Rest things looks good you can merge this branch

File: src/pages/SignUp/SignUp.jsx (hunk 1 above)
Comment (anjali1102) [reply src/pages/SignUp/SignUp.jsx:12]: will do later

**Pull request merged**